apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/edit/components/RatingQuestionForm.tsx

```diff
@@ -16,62 +17,79 @@ interface RatingQuestionFormProps {
   questionIdx: number;
   updateQuestion: (questionIdx: number, updatedAttributes: any) => void;
   lastQuestion: boolean;
+  selectedLanguage: string;
+  setSelectedLanguage: (language: string) => void;
+  languages: string[][];
   isInvalid: boolean;
 }
 
 export default function RatingQuestionForm({
   question,
   questionIdx,
   updateQuestion,
-  lastQuestion,
   isInvalid,
   localSurvey,
+  selectedLanguage,
+  setSelectedLanguage,
+  languages,
 }: RatingQuestionFormProps) {
   const [showSubheader, setShowSubheader] = useState(!!question.subheader);
-  const environmentId = localSurvey.environmentId;
 
   return (
     <form>
-      <QuestionFormInput
+      <LocalizedInput
+        id="headline"
+        name="headline"
+        value={question.headline as TI18nString}
```

Comment:
Also here we need to be cautious with the type conversions. As this is ignoring typescript. We should a better way.